ops/inventories/dev-docker.yml

```diff
@@ -0,0 +1,19 @@
+all:
```

Comment:
Can you find a way to avoid having two dev inventory file? It seems the only difference is the ssh key path.

Reply:
it's done, but I couldn't make it work without using a password when connecting to `vagrant ssh`